core/server/data/meta/context_object.js

```diff
@@ -1,8 +1,16 @@
-var config = require('../../config'),
-    _      = require('lodash');
+var settingsCache = require('../../api/settings').cache,
+    _ = require('lodash');
 
 function getContextObject(data, context) {
-    var blog = config.get('theme'),
+    /**
+     * @TODO:
+     *   - the only valid property is cover
+     *   - there was no image property in config.get('theme')
+     *   - there was no author property in config.get('theme')
+     */
```

Comment:
From what I see, this was a fallback option. It's possibly a  product of the refactoring that has gone on. I'm not sure that cover is the only valid option - what about title and logo?

Again, I think all of this code is half-refactored.